schemas/devContainerTemplate.schema.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Development Container Template Metadata",
+  "description": "Development Container Template Metadata (devcontainer-template.json). See https://containers.dev/implementors/templates/ for more information.",
+  "definitions": {
+    "Template": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "$schema": {
```

Comment:
This would also make sense in the features schema.

Reply:
Will do.

Reply:
I just pushed a new commit to the PR which adds this.